src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,57 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    private static int ZERO = 0;
```

Comment:
`ZERO` 또한 `final` 키워드까지 추가하여 상수로 선언해보면 어떨까요?
```suggestion
    private static final int ZERO = 0;
```